.circleci/config.yml

```diff
@@ -1,81 +1,58 @@
 version: 2.1
-executors:
-  default:
-    working_directory: /go/src/github.com/mattermost/mattermost-plugin-github
-    docker:
-      - image: circleci/golang:1.12-node
 
+orbs:
+  plugin-ci: mattermost/plugin-ci@volatile
```

Comment:
Out of interest: Is there a repo for this?

Reply:
Soon.

Reply:
https://github.com/mattermost/circleci-orbs